testcontainers-dapr/src/main/java/io/dapr/testcontainers/DaprContainer.java

```diff
@@ -70,9 +72,11 @@ public class DaprContainer extends GenericContainer<DaprContainer> {
   private Integer appPort;
   private String appHealthCheckPath;
   private boolean shouldReusePlacement;
```

Comment:
It is not clear, why we need this change. As far as I saw in the IT tests, the network alias is not required. I am not against enhancing `DaprContainer` with network alias, but I think it can be done in a separate PR where we could explain what is the use case and probably and a few unit tests to improve coverage.

Reply:
@mcruzdev I see that you are using `.withNetwork(NETWORK)` for `ToxiproxyContainer`, hence my question regarding network alias.

As I mentioned we can add support for network alias in a separate PR and add the necessary code coverage to `DaprContainer`. Once we have support for network alias we can revisit this IT and use network alias instead of network.